jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/DefaultServlet.java

```diff
@@ -191,6 +189,7 @@ public void init() throws ServletException
                     LOG.warn("Unable to use stylesheet: {} - {}", stylesheetParam, e.toString());
             }
         }
+        _resourceService.loadDefaultStylesheetIfNotPresent();
```

Comment:
If `ResourceService` was a LifeCycle, this could be done in the `doStart()` method instead.